api/client_rest_test.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestRESTGetDelete(t *testing.T) {
```

Comment:
Could this test be in `api/client_test.go` instead of a new file?

Reply:
Certainly can.